.dots/VibeBots-f-240-make-45187b28.html

```
@@ -0,0 +1,20 @@
+<!doctype html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)